.github/workflows/build.yml

```diff
@@ -30,7 +36,7 @@ jobs:
         run: make
 
       - name: Deploy
-        uses: JamesIves/github-pages-deploy-action@4.1.4
+        uses: JamesIves/github-pages-deploy-action@4.3.4
```

Comment:
Hi @cclauss! Do you have idea why this deploy might have failed?

https://github.com/psycopg/psycopg-website/runs/7108663001?check_suite_focus=true

The release number seems right.